### ISSUE TITLE
Ionic doctor didn't find an interesting error

### ISSUE THREAD

**Issue by tryadelion** [ionic-team/ionic-cli#2980]
was launching an old project (2 weeks old) and ran onto this issue:

![captura de pantalla 2018-02-08 a las 8 31 39](https://user-images.githubusercontent.com/3778474/35960632-b9684c78-0caa-11e8-889a-3cdc2f98cac4.png)

That seems quite odd, shouldn't missing files return failure exit code to the doctor check script?

**Comment by dwieeb** [comment]
@tryadelion Not necessarily. It's not a fatal error. What's your `ionic info` for that project?
